lib/permutations.rb

```diff
@@ -1,4 +1,29 @@
 
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
+  # raise NotImplementedError, "permutations? not implemented"
+
+  if string1 == "" && string2 == ""
+    return true
+  end
+
+  if string1.length != string2.length
+    return false
+  end
+
+  new_hash = {}
+  i = 0
+  while i < string1.length
+    new_hash[string1[i]] = true
+    i += 1
+  end
+
+  i = 0
+  while i < string2.length
+    if new_hash[string2[i]] == true
+      i += 1
+    else
+      return false
+    end
```

Comment:
While this satisfies the tests, it misses that the string lengths could be the same, but the quantities of each letter might be different.  Consider `feed` and `fedd`.